docs/decompiling.md

```diff
@@ -0,0 +1,30 @@
+# Decompiling an ARM Template
+ 
+> Requires Bicep CLI v0.2.59 or later
+ 
+The Bicep CLI provides the ability to decompile any existing ARM Template to a `.bicep` file, using the `decompile` command:
+```sh
+bicep decompile "path/to/file.json"
+```
+
+You can use this command to get to a starting point for Bicep authoring. Note that because there is no guaranteed conversion from JSON to Bicep, decompilation may fail, or you may be left with errors/warnings in the generated Bicep file to fix up. See [Limitations](#limiations) for some details of what is not currently possible.
+
+## Exporting a resource group
+You can pass an exported template directly to the `bicep decompile` command to effectively export a resource group to a `.bicep` file.
```

Comment:
Would link to docs on export as well (which I assume we have?), if people want to export via the portal or want more details